js/lib/Country.js

```diff
@@ -80,9 +80,15 @@ var Country = function (options) {
 	}
 
 	this.locale = this.locale || new Locale();
-	localeinfo = new LocaleInfo(this.locale);
-
-	if (localeinfo.getRegionName() === undefined) {
+	new LocaleInfo(this.locale, {
+		sync: sync,
+		loadParams: loadParams,
+		onLoad: ilib.bind(this, function (li) {
+			this.locinfo = li;
+		})
+	});
+
+	if (this.locinfo.getRegionName() === undefined) {
```

Comment:
The rest of this constructor has to appear inside of the onLoad callback on line 86 or else this.locinfo will be undefined on line 91 during async operation. The onLoad function will be called when the new LocaleInfo is created, which may occur after this constructor is finished running.